packages/web/e2e/uploadTrack.test.ts

```diff
@@ -169,6 +169,7 @@ test('should upload a remix, hidden, AI-attributed track', async ({ page }) => {
 })
 
 test('should upload a premium track', async ({ page, browser }) => {
+  test.setTimeout(20000)
```

Comment:
This changes the timeout for the test as a whole. I think you want a different timeout for this expect assertion based on what you were saying earlier:
https://github.com/AudiusProject/audius-protocol/pull/12160/files#diff-04ad3b4f3f77637c9a0f5fd16303e5c10f1537df405c7d1005ddfd7c859b97f6R224

https://playwright.dev/docs/test-timeouts